scripts/test

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${NASA_HSI_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Run tests.
+"
+}
+
+if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
+    if [[ "${1:-}" == "--help" ]]; then
+        usage
+    else
+        # Lint Bash scripts
+        docker-compose \
+            run --rm --no-deps shellcheck \
+            scripts/*
+
+        # Lint Python
+        # docker-compose \
+        #     run --rm --no-deps \
+        #     black --check --diff .
```

Comment:
This is commented out because this produced a fairly large diff. I'd like to merge it as is and create an issue for one of the application developers to investigate the formatting issues.